README.md

```diff
@@ -189,7 +189,7 @@ These parameters take precedence over the file on disk if both are specified.
 
 To access Grafana, use the following in your browser of choice: 
 ```http://[AGENT IP ADDRESS]/3000```
-
+Make sure to open the port 3000 as this is not opened by default.
```

Comment:
This is more of an integrations task, not instructional from the module point of view